cdap-proto/src/main/java/co/cask/cdap/proto/PluginDetail.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.proto;
+
+/**
+ * Represents a plugin id and name in an HTTP response.
+ */
+public class PluginDetail {
+  private final String pluginId;
```

Comment:
'plugin' prefix is redundant, can just use id and name. Probably worth having the type in here too.